config/config.go

```diff
@@ -41,15 +42,30 @@ func Get() *SuperKeyWorkerConfig {
 		options.SetDefault("AwsRegion", cfg.Logging.Cloudwatch.Region)
 		options.SetDefault("AwsAccessKeyId", cfg.Logging.Cloudwatch.AccessKeyId)
 		options.SetDefault("AwsSecretAccessKey", cfg.Logging.Cloudwatch.SecretAccessKey)
-		options.SetDefault("KafkaBrokers", []string{fmt.Sprintf("%s:%v", cfg.Kafka.Brokers[0].Hostname, *cfg.Kafka.Brokers[0].Port)})
+		options.SetDefault("KafkaBrokerConfig", cfg.Kafka.Brokers[0])
 		options.SetDefault("LogGroup", cfg.Logging.Cloudwatch.LogGroup)
 		options.SetDefault("MetricsPort", cfg.MetricsPort)
 
 	} else {
 		options.SetDefault("AwsRegion", "us-east-1")
 		options.SetDefault("AwsAccessKeyId", os.Getenv("CW_AWS_ACCESS_KEY_ID"))
 		options.SetDefault("AwsSecretAccessKey", os.Getenv("CW_AWS_SECRET_ACCESS_KEY"))
-		options.SetDefault("KafkaBrokers", []string{fmt.Sprintf("%v:%v", os.Getenv("QUEUE_HOST"), os.Getenv("QUEUE_PORT"))})
+
+		kafkaPort := os.Getenv("QUEUE_PORT")
+		if kafkaPort != "" {
+			port, err := strconv.Atoi(kafkaPort)
+			if err != nil {
+				log.Fatalf(`the provided "QUEUE_PORT", "%s", is not a valid integer: %s`, kafkaPort, err)
+			}
+
+			brokerConfig := &clowder.BrokerConfig{
+				Hostname: os.Getenv("QUEUE_HOST"),
+				Port:     &port,
+			}
```

Comment:
This should probably not be a pointer per https://github.com/RedHatInsights/sources-api-go/pull/422 right?

Reply:
Exactly, yes :+1: